src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -200,6 +200,12 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.SanitizeSqlStatements"/>
         public const string SanitizeSqlStatements = "SIGNALFX_SANITIZE_SQL_STATEMENTS";
 
+        /// <summary>
+        /// Comma-separated list of DiagnosticListeners to subscribe to ASP.NET instrumentation DiagnosticObserver
+        /// </summary>
+        /// <seealso cref="TracerSettings.AdditionalDiagnosticListeners"/>
+        public const string AdditionalDiagnosticListeners = "SIGNALFX_INSTRUMENTATION_ASPNETCORE_ADDITIONAL_DIAGNOSTIC_LISTENERS";
```

Comment:
Could we go with something shorter? or does it need to match some existing patterns?
```suggestion
        public const string AdditionalDiagnosticListeners = "SIGNALFX_ASPNETCORE_DIAGNOSTIC_LISTENERS";
```

Reply:
The `SIGNALFX_INSTRUMENTATION_` prefix denotes it's only applicable to the specified instrumentation and has been used for java as well.   Is `SIGNALFX_INSTRUMENTATION_ASPNETCORE_DIAGNOSTIC_LISTENERS` ok?

Reply:
Yes, dropping "ADDITIONAL" is an improvement.